wafer/talks/views.py

```diff
@@ -63,7 +65,18 @@ def get_object(self, *args, **kwargs):
 
     def get_context_data(self, **kwargs):
         context = super(TalkView, self).get_context_data(**kwargs)
-        context['can_edit'] = self.object.can_edit(self.request.user)
+        talk = self.object
+        user = self.request.user
+
+        context['can_edit'] = talk.can_edit(user)
+
+        can_review = Review.can_review(user)
+        context['can_review'] = can_review
+        if can_review:
+            review = talk.reviews.filter(reviewer=user).first()
+            context['review_form'] = ReviewForm(
+                instance=review, talk=talk, user=user)
```

Comment:
For many smaller conferences, the pool of speakers and the pool of reviewers is going to overlap.
Should we explicitly check that that the reviewer is not an author?

Reply:
Done in 4490164